corehq/apps/users/views/__init__.py

```diff
@@ -954,7 +954,7 @@ def _update_role_from_view(domain, role_data):
         not domain_has_privilege(domain, privileges.CUSTOM_DOMAIN_ALERTS)
         and 'manage_domain_alerts' in role_data['permissions']
     ):
-        raise ValueError(_("Update subscription to set access for custom domain alerts"))
+        role_data['permissions']['manage_domain_alerts'] = False
```

Comment:
Hey @zandre-eng 

thanks for the fix.

The error was added simply to block the update because the project that does not have access to the feature, should not be updating the value for this permission. It seems there is a miss there. Are you able to clearly point out where the miss is?

As for the update,
I'd say we don't modify the permission if the domain does not have privilege to do so and retain whatever it is in DB as it is because
1. We should not edit domain's data/settings
2. If the domain regains their access to the feature, they would want the value they set still be available and not be reset, which would take them by surprise.

Reply:
Thanks @zandre-eng this sounds good to me.

Appreciate the follow up. I request that the description get updated accordingly with details. The related PR flagged in the description should be the one that GA'ed the feature.